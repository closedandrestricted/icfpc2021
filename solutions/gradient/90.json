{"vertices":[[377,491],[286,514],[289,481],[400,551],[334,524],[505,565],[318,596],[362,523],[432,544],[332,514],[299,631],[343,574],[431,576],[421,519],[434,555],[409,513],[342,539],[413,551],[362,521],[364,629],[388,592],[392,533],[354,651],[417,583],[421,538],[412,517],[399,498],[371,526],[370,564],[315,503],[360,563],[334,568],[399,504],[333,516],[350,547],[402,474],[424,532],[402,539],[361,556],[310,583],[385,649],[350,559],[402,587],[432,471],[348,522],[414,571],[407,534],[396,539],[403,513],[378,584],[403,672],[325,595],[379,472],[390,534],[458,561],[361,614],[422,590],[419,584],[349,471],[374,537],[376,591],[398,584],[490,643],[364,523],[404,573],[375,471],[422,524],[402,548],[424,536],[323,497],[348,638],[464,737],[306,563],[363,546],[434,447],[389,485],[414,475],[391,620],[412,478],[359,519],[457,499],[378,589],[388,475],[405,407],[398,672],[389,580],[423,518],[425,701],[367,529],[411,604],[403,588],[317,450],[401,543],[306,545],[432,534],[370,515],[342,528],[404,516],[378,786],[408,462],[376,402],[390,477],[407,585],[320,501],[451,617],[352,637],[450,540],[455,734],[412,672],[370,455],[372,403],[361,521],[373,465],[320,705],[337,597],[428,595],[434,528],[395,589],[320,469],[354,330],[365,530],[422,652],[395,878],[382,566],[422,561],[451,734],[323,395],[414,467],[472,830],[482,680],[398,539],[381,555],[288,402],[431,594],[358,592],[365,405],[396,385],[479,616],[415,685],[329,463],[475,918],[457,632],[470,507],[328,485],[398,529],[320,469],[404,475],[390,515],[434,548],[354,586]]}