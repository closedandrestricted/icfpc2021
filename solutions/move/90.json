{"vertices":[[257,427],[167,452],[321,487],[279,490],[213,463],[378,523],[197,535],[241,462],[312,484],[211,453],[178,570],[222,513],[310,515],[300,458],[313,494],[288,452],[221,478],[292,490],[241,460],[243,568],[267,531],[271,472],[233,590],[296,522],[300,477],[291,456],[278,437],[250,465],[249,503],[194,442],[239,502],[213,507],[278,443],[212,455],[229,486],[281,413],[303,471],[281,478],[240,495],[189,522],[264,588],[229,498],[281,526],[311,410],[227,461],[293,510],[287,473],[275,478],[282,452],[257,523],[282,611],[204,534],[258,411],[268,473],[337,500],[240,553],[301,529],[298,523],[228,410],[253,476],[255,530],[277,523],[368,582],[243,462],[283,512],[254,410],[301,463],[281,487],[303,475],[202,436],[227,577],[342,676],[185,502],[242,485],[313,386],[268,424],[293,414],[270,559],[291,417],[238,458],[336,438],[257,528],[267,414],[283,346],[277,611],[268,519],[302,457],[304,640],[246,468],[290,543],[282,527],[196,389],[280,482],[185,484],[311,473],[249,454],[221,467],[283,455],[257,725],[287,401],[255,341],[269,416],[286,524],[199,440],[329,556],[231,576],[329,479],[334,673],[291,611],[249,394],[251,342],[240,460],[252,404],[200,642],[216,536],[307,534],[313,467],[274,528],[199,408],[232,269],[244,469],[301,591],[274,817],[260,504],[301,500],[330,673],[202,334],[293,406],[350,769],[361,619],[277,478],[260,494],[173,338],[310,533],[237,531],[244,344],[275,324],[358,555],[294,624],[208,402],[354,856],[336,571],[357,464],[207,424],[277,468],[199,408],[283,414],[269,454],[313,487],[233,525]]}