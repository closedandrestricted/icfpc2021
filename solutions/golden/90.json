{"vertices":[[377,467],[286,492],[441,527],[399,530],[333,503],[502,554],[317,575],[361,502],[432,524],[331,493],[298,610],[342,553],[430,555],[420,498],[433,534],[408,492],[341,518],[412,530],[361,500],[363,608],[387,571],[391,512],[353,630],[416,562],[420,517],[411,496],[398,477],[370,505],[369,543],[314,482],[359,542],[333,547],[398,483],[332,495],[349,526],[401,453],[423,511],[401,518],[360,535],[309,562],[384,628],[349,538],[401,566],[431,450],[347,501],[413,550],[407,513],[395,518],[402,492],[377,563],[402,651],[324,574],[378,451],[388,513],[457,540],[360,593],[421,569],[418,563],[348,450],[373,516],[375,570],[397,563],[489,622],[363,502],[403,552],[374,450],[421,503],[401,527],[423,515],[322,476],[347,617],[462,716],[305,542],[362,525],[433,426],[388,464],[413,454],[390,599],[411,457],[358,498],[456,478],[377,568],[387,454],[403,386],[397,651],[388,559],[422,497],[424,680],[366,508],[410,583],[402,567],[316,429],[400,522],[305,524],[431,513],[369,494],[341,507],[403,495],[377,765],[407,441],[375,381],[389,456],[406,564],[319,480],[449,596],[351,616],[449,519],[454,713],[411,651],[369,434],[371,382],[360,500],[372,444],[319,684],[336,576],[427,574],[433,507],[394,568],[319,448],[353,309],[364,509],[421,631],[394,857],[380,544],[421,540],[450,713],[322,374],[413,446],[471,809],[481,659],[397,518],[380,534],[287,381],[430,573],[357,571],[364,384],[395,364],[478,595],[414,664],[328,442],[475,896],[456,611],[474,495],[327,464],[397,508],[319,448],[403,454],[389,494],[433,527],[353,565]]}